addons/sms/wizard/sms_composer_views.xml

```diff
@@ -5,6 +5,24 @@
         <field name="model">sms.composer</field>
         <field name="arch" type="xml">
             <form string="Send an SMS">
+                <!-- Single mode information (invalid number) -->
+                <div colspan="2" class="alert alert-danger text-center mb-3" role="alert"
+                    attrs="{'invisible': ['|', ('comment_single_recipient', '=', False), ('recipient_single_valid', '=', True)]}">
+                    <p class="my-0">
+                        <span>Invalid number, make sure to set a country on the </span>
+                        <span><field name="res_model_description" readonly="1"/></span>
```

Comment:
Also, don't think readonly needs to be specified as this is a regular compute field.